setup.py

```diff
@@ -13,16 +13,15 @@
 ]
 
 install_requires = [
-    "numpy>=1.13.0",
-    "six==1.11.0"
+    "scikit-learn~=1.0",
 ]
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 setup(
     name="scikit-spark",
-    version="0.4.0",
+    version="0.5.0",
```

Comment:
Let's release this as a major version now so in case we make any future breaking changes we can increment the major version again. The interface at least is probably not going to change much now after this has been in use for so long.

Reply:
:heavy_check_mark: